index.php

```diff
@@ -0,0 +1,28 @@
+<?php
+require_once("src/IPublicable.php");
+require_once("src/Autor.php");
+require_once("src/libro.php");
+require_once("src/revista.php");
+
+$autor1 = new Autor("Elena G. de White", "Americana");
+$autor2 = new Autor("Gabriel García Márquez", "Colombiano");
+$autor3 = new Autor("Julio Cortázar", "Argentino");
+$autor4 = new Autor("Mario Vargas Llosa", "Peruano");
```

Comment:
```suggestion
$autorElenaWhite = new Autor("Elena G. de White", "Americana");
$autorGabrielGarcia = new Autor("Gabriel García Márquez", "Colombiano");
$autorJulioCortazar = new Autor("Julio Cortázar", "Argentino");
$autorMarioVargas = new Autor("Mario Vargas Llosa", "Peruano");
```
usar variables con nombres descriptivos